### PR TITLE
Adjust JSONField to include widget name

### PR DESCRIPTION
**Pull request by sneridagh**

### CODE REVIEW

File: plone/schema/jsonfield.py [hunk 1]
```diff
@@ -17,27 +17,24 @@
...
 class JSONField(Field):
-
-    def __init__(self, schema=DEFAULT_JSON_SCHEMA, **kw):
+    def __init__(self, schema=DEFAULT_JSON_SCHEMA, widget=None, **kw):
         if not isinstance(schema, str):
             raise WrongType
+        if widget and not isinstance(widget, str):
+            raise WrongType
+
+        self.widget = widget
```
Comment (tiberiuichim) [inline plone/schema/jsonfield.py:37]: Maybe go with "widget_name" ?

Comment (sneridagh) [reply plone/schema/jsonfield.py:37]: mmm the serialisation already uses the "widget" naming (both in the adapters and in the property name)... :( being in the Python side, if no class is going to be specified there, ever... is it not too much specialisation?

**Review by avoinea — APPROVED** [review]

**Comment by sneridagh** [comment]
@jenkins-plone-org please run jobs

**Review by tiberiuichim — APPROVED** [review]

**Comment by sneridagh** [comment]
@tisto ready, related to https://github.com/plone/plone.restapi/pull/1091

**Pull request merged**